optuna_dashboard/app.py

```diff
@@ -94,6 +95,13 @@ def get_trials(
     return trials
 
 
+def get_distribution_name(param_name: str, study: Study) -> str:
+    for trial in study.trials:
+        if param_name in trial.distributions:
+            return trial.distributions[param_name].__class__.__name__
+    assert False
```

Comment:
Could you add a message?

```suggestion
    assert False, "Must not reach here."
```